### PR TITLE
netcoreapp2.0

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: test/Microsoft.AspNetCore.Rewrite.Tests/Microsoft.AspNetCore.Rewrite.Tests.csproj [hunk 1]
```diff
@@ -3,8 +3,7 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```
Comment (kichalla) [inline test/Microsoft.AspNetCore.Rewrite.Tests/Microsoft.AspNetCore.Rewrite.Tests.csproj]: you can remove the following 2 elements cc @pranavkm

Comment (kichalla) [inline test/Microsoft.AspNetCore.Rewrite.Tests/Microsoft.AspNetCore.Rewrite.Tests.csproj]: There seem to be other files with similar content.

File: src/Microsoft.AspNetCore.Buffering/BufferingWriteStream.cs [hunk 2]
```diff
@@ -160,34 +160,7 @@ public override Task WriteAsync(byte[] buffer, int offset, int count, Cancellati
                 return _innerStream.WriteAsync(buffer, offset, count, cancellationToken);
             }
         }
-#if NET46
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Buffering/BufferingWriteStream.cs]: Keep these APIs

File: src/Microsoft.AspNetCore.ResponseCompression/CompressionProviderCollection.cs [hunk 3]
```diff
@@ -3,9 +3,7 @@
 
 using System;
 using System.Collections.ObjectModel;
-#if NETSTANDARD1_3
 using System.Reflection;
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.ResponseCompression/CompressionProviderCollection.cs]: Remove

File: src/Microsoft.AspNetCore.ResponseCompression/BodyWrapperStream.cs [hunk 4]
```diff
@@ -137,58 +137,6 @@ public override void Write(byte[] buffer, int offset, int count)
             }
         }
 
-#if NET46
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.ResponseCompression/BodyWrapperStream.cs]: Revert

File: test/Microsoft.AspNetCore.ResponseCompression.Tests/BodyWrapperStreamTests.cs [hunk 5]
```diff
@@ -108,36 +108,6 @@ public void Write_IsPassedToUnderlyingStream_WhenDisableResponseBuffering(bool f
             Assert.Equal(File.ReadAllBytes(path), written);
         }
 
-#if NET46
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.ResponseCompression.Tests/BodyWrapperStreamTests.cs]: Revert

File: test/Microsoft.AspNetCore.ResponseCompression.Tests/ResponseCompressionMiddlewareTest.cs [hunk 6]
```diff
@@ -570,16 +563,9 @@ public void Options_HttpsDisabledByDefault()
 
             var response = await client.SendAsync(request, HttpCompletionOption.ResponseHeadersRead);
 
-#if NET46 // Flush not supported, compression disabled
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.ResponseCompression.Tests/ResponseCompressionMiddlewareTest.cs]: Does this work?

Comment (pakrym) [inline test/Microsoft.AspNetCore.ResponseCompression.Tests/ResponseCompressionMiddlewareTest.cs]: Flush is supported on core

**Comment by pakrym** [comment]
@pranavkm :up::date:

**Pull request merged**